llvm/test/lab2/ionova_ekaterina/test.ll

```diff
@@ -0,0 +1,38 @@
+; RUN: opt -load-pass-plugin %llvmshlibdir/Instrument_Function_Ionova_Ekaterina_FIIT2%pluginext\
+; RUN: -passes=instrumentation_function -S %s | FileCheck %s
+
```

Comment:
Please add a test for case when a function already has instrument_start/instrument_end calls.